test/BaselineOutput/SingleDebug/SDCA/BinarySDCA-SmoothedHinge-CV-breast-cancer-out.txt

```diff
@@ -3,12 +3,14 @@ Automatically adding a MinMax normalization transform, use 'norm=Warn' or 'norm=
 Using 1 thread to train.
 Warning: Skipped 8 instances with missing features/label during training
```

Comment:
Since you touching baseline files. Any chance you want to move them to Common folder and delete Release and Debug versions? Unless they actually different in Release and Debug

Reply:
Those changes will be discard for having BC so I will have no plan to touch those files.

---
In reply to: [256211363](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256211363) [](ancestors = 256211363)